app/ts/utils/ethereumUtils.ts

```diff
@@ -76,6 +76,17 @@ export function bigintToDecimalStringWithUnknown(value: bigint | undefined, powe
 	return value === undefined ? '?' : bigintToDecimalString(value, power, maxDecimals)
 }
 
+export const isPracticallyInfinite = (value: bigint, power: bigint) => {
+	const googol = 10n ^ 100n
+	return value / (10n ** power) > googol
+}
```

Comment:
I think this is actually more than infinite.  `log10(2**256) ~= 77`, so I don't think we can actually get this big.  `2^250` seems fine, or `10^60` if you prefer decimal.